OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -4,13 +4,15 @@
 
 iq:
   name: nxiq
-  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server
-  imageTag: 1.85.0-01-ubi
+  imageName: sonatype/nexus-iq-server
+  imageTag: 1.91.0
```

Comment:
if we're going to upgrade this now, how do we feel about combining the version into imageName?  The operator requires it to be combined, but does it go against a rival best-practice for helm?

Reply:
we had kept them separate because the build process would try to hunt out the version and update it automatically. It may be possible to combine it, but we'd need to update the Jenkinsfile and it will likely introduce some more complexity there. Definitely a trade-off.

Reply:
I probably shouldn't have tried to lump that combination into this PR, so you're right, let's not try to combine name and tag just yet.

Reply:
For helm, the repo, imagename, and tags should all be separate entries so they can be ovrridden